src/components/Disputes/actions/DisputeAppeal.js

```diff
@@ -24,8 +24,22 @@ function DisputeAppeal({ onRequestAppeal, confirm }) {
       >
         {actionLabel}
       </Button>
-      <Info mode="description">
-        Anyone holding the required appeal deposit can trigger this action.
+      <Info>
+        <strong>Anyone</strong> can{' '}
+        <strong>
+          lock DAI as collateral to{' '}
+          {confirm ? 'confirm an appeal' : 'propose an appeal'}{' '}
+        </strong>
+        , if they don’t agree with the current outcome. When the final ruling is
+        confirmed, the user who {confirm ? 'confirmed the appeal' : 'appealed'}{' '}
+        gets rewarded if the ruling has switched in their favor. If not, their
+        entire collateral could be re-distributed to the winning party.{' '}
+        {confirm
+          ? 'If an appeal is confirmed, a new adjudication round is initiated and a new jury is drafted.'
```

Comment:
```suggestion
          ? 'If an appeal is confirmed, a new adjudication round is initiated and a new jury is drafted. '
```

Although in this situation, I think having this line in both cases is useful; it's just switching the "If" for a "When".

Reply:
Yes, good suggestion